sockeye/train.py

```diff
@@ -287,6 +259,8 @@ def create_data_iters_and_vocab(args: argparse.Namespace,
     batch_num_devices = 1 if args.use_cpu else sum(-di if di < 0 else 1 for di in args.device_ids)
     batch_by_words = args.batch_type == C.BATCH_TYPE_WORD
 
+    validation_sources = list(map(str, map(os.path.abspath, [args.validation_source] + args.validation_source_factors)))
```

Comment:
lots of `map`s popping up. Wouldn't a list comprehension be clearer? 
```
validation_sources = [args.validation_source] + args.validation_source_factors
validation_sources = [str(os.path.abspath(source)) for source in validation_sources]
```

Reply:
yes